lightning/src/ln/channelmanager.rs

```diff
@@ -1551,10 +1553,33 @@ impl ChannelDetails {
 			inbound_htlc_minimum_msat: Some(context.get_holder_htlc_minimum_msat()),
 			inbound_htlc_maximum_msat: context.get_holder_htlc_maximum_msat(),
 			config: Some(context.config()),
+			channel_shutdown_state: Some(context.shutdown_state()),
 		}
 	}
 }
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+/// Further information on the details of the channel shutdown.
+/// Upon channels being forced closed, (ie. funding tx spend confirmed detected by chain monitor)
+/// ChannelShutdownState will be set into ShutdownComplete state or the channel will
+/// be shortly removed
```

Comment:
just grammar/ticks nitting because it's public docs: 
```suggestion
/// `ChannelShutdownState` will be set to `ShutdownComplete` or the channel will
/// be removed shortly.
```